dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tei/TeiQueryController.java

```diff
@@ -104,7 +109,9 @@
     @Nonnull
     private final ContextUtils contextUtils;
 
-    @GetMapping( "query/{trackedEntityType}" )
+    @GetMapping( value = "query/{trackedEntityType}", produces = { APPLICATION_JSON_VALUE,
+        "application/javascript" } )
+    @ResponseBody
```

Comment:
When the class is annotated with @RestController, then @ResponseBody is implicit